src/stores/views.ts

```diff
@@ -713,7 +734,7 @@ export function viewAddFilterItemsToRow(
         if (r.id === row.id) {
           returnRow = r;
           const newLayers = r.layers.map(l => {
-            if (l.id === layer.id) {
+            if (layer !== undefined && l.id === layer.id) {
```

Comment:
This check may be tied to my comment above

Reply:
Reverted to the `develop` version as it also should not be needed with the rework to external event filtering!